src/lookup-table/services/LookupTable.service.ts

```diff
@@ -207,6 +213,21 @@ const LookupTableService = async (props: LookupTableServiceProps) => {
     }
   }
 
+  const createOrUpdateDataView = async () => {
+    const existingDataViews = await lookupTableModel.getDataViews(
+      tableName,
+      structure?.dataViewCode ?? dataViewCode
+    )
+
+    if (existingDataViews.length > 0) {
+      for (const dataView of existingDataViews) {
+        await lookupTableModel.updateDataView(dataView.id, name, dataViewCode)
```

Comment:
As mentioned above, the dataViewCode can be changed -- it gets passed into this Service object when instantiated by the endpoint, so it's in "global" scope for this service rather than being an input parameter to this function. 

(I don't particularly like this pattern, but can't really refactor the whole thing right now)